Minor grammar change and remove outdated reference

paulkel opened the pull request:
Removed outdated reference to future release of Azure SDK (this was released more than a year ago) and fixed one grammatical error.

azurecla commented:
Hi __@KundanaP__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

tysonn commented:
pm

tysonn commented:
Tyson handling

paulkel committed:
Minor grammar change and remove outdated reference

Removed outdated reference to future release of Azure SDK (this was released more than a year ago) and fixed one grammatical error.
